OWNERS

```diff
@@ -6,8 +6,13 @@ approvers:
  - furkatgofurov7
 
 reviewers:
- - macaptain
+ - smoshiur1237
+ - Rozzi
```

Comment:
This is your own GitHub handle @Rozzii :grinning:
```suggestion
 - Rozzii
```

Reply:
Thanks!